README.md

```diff
@@ -27,11 +27,19 @@ individual tasks.
 
 # Installation
 
-Run is a single binary, which you can download from from the [releases
-page][releases].
+Run is a single binary, which you can download from from the [releases page][releases].
 
-Alternately, if you already use go, you can install Run with the go command
-line tool:
+## Install Script (MacOS, Linux)
+
+Download and install the latest version of `run` to the current working directory.
```

Comment:
this reads a bit like an instruction, rather than a description of the install script. Please make it clearer.

(one idea: a prefix like "this command will…")